cpp/dolfin/fem/Form.h

```diff
@@ -123,6 +127,26 @@ class Form
   ///         coefficients.
   int original_coefficient_position(int i) const;
 
+  /// Set constants based on their names
+  ///
+  /// This method is used in command-line workflow, when users set
+  /// constants to the form in cpp file.
+  ///
+  /// Names of the constants must agree with their names in UFL file.
+  void set_constants(
```

Comment:
These set functions are confusing given that the Constructor takes a constants argument.

Reply:
As said above, construction of Form takes places without full knowledge about Constants (and Functions). Just later, user sets the constants and functions. What can I do to avoid the confusion?